### DISCUSSION TITLE
Please add free dynv6.com as DNS Provider

### DISCUSSION THREAD

**Discussion by indiana11011100** [NginxProxyManager/nginx-proxy-manager#2488] [Polls]
I use dynv6.com as my DynDNS provider with my FRITZ!Box. Of course there maybe others but there are some things which I think would make it really nice to be implemented also in Nginx Proxy Manager.
1. It is free, stable and easy to use.
2. It probably will exist longer than other small private projects.
3. There is no other way to make a wildcard challenge for local subdomains.
I wanna use ports for public services. So the domain will stay the same. For local services / devices and much better remembering names than IP's it would be really nice to use the same certificate :)
4. Maybe there is another way to implement it manually but I havn't found. In this case please give me a hint.

At least I wanna say I appreciate your work a lot as it makes it so much easier to deal with nginx for all of us!

greets indy

Sorry, I should have read a bit more RFC 2136 does the wildcard certificate :)

**Comment by freeztyler** [comment]
I'm currently struggling with the problem you had. Can you please tell me what values ​​you entered for dynv6 so that it runs?

# Target DNS server
dns_rfc2136_server = 185.55.116.154  <--- ?
# Target DNS port
dns_rfc2136_port = 53 <---?
# TSIG key name
dns_rfc2136_name = tsig-XXXXXXXX.dynv6.com
# TSIG key secret
dns_rfc2136_secret = XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX==
# TSIG key algorithm
dns_rfc2136_algorithm = HMAC-SHA512

i get a timeout :(

**Reply by indiana11011100** [reply]
Moin freeztyler,

seems your server is address is different:
# Target DNS server (ns1.dynv6.com - Nginx Proxy Manager v2.x handles IP only)
dns_rfc2136_server = 95.216.144.82
# Target DNS port (default 53)
dns_rfc2136_port = 53
# TSIG key name (Must be able to be genereated from DynDNS Provider)
dns_rfc2136_name = YourKeyName!
# TSIG key secret (The generated secret)
dns_rfc2136_secret = YourSecret!
# TSIG key algorithm (Algorithm chosen or available)
dns_rfc2136_algorithm = HMAC-SHA512

I did a simple ping ns1.dynv6.com to resolve the IP.

**Reply by freeztyler** [reply]
thx, it works!
